### ISSUE TITLE
Linux build failed

### ISSUE THREAD

**Issue by pelson** [conda-forge/pyzmq-feedstock#2]
```
===== pyzmq-14.7.0-py27_0 OK =====
+ nosetests zmq.tests
..............F..SSSSSSSSSSSSSSSS.....................S.................................S.......S......................................S...S...........................S.................
======================================================================
FAIL: threaded auth - NULL
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/conda/envs/_test/lib/python2.7/site-packages/zmq/tests/test_auth.py", line 127, in test_null
    self.assertTrue(self.can_connect(server, client))
AssertionError: False is not true
-------------------- >> begin captured logging << --------------------
zmq.auth: DEBUG: auth received API command 'TERMINATE'
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 185 tests in 27.057s
```

This wasn't happening on staged-recipes...

**Comment by jakirkham** [comment]
I restarted the build on Circle CI and it passed. I tried to reproduce this locally and was unable. Just to ensure it wasn't due to changes I made to the container, I tried the old image and new image on Docker Hub and both passed.

There have been issues with ZeroMQ (and possibly other things) in Docker containers before. See this [comment]( https://github.com/ipython/ipython/issues/7062#issuecomment-64973280 ). Though I don't believe it was ever narrowed down to ZeroMQ specifically. In any event, using `tini` has not seen similar issues and has been quite robust so far. So maybe this has fixed by the Docker container updates.

That all being said, I'm not exactly sure what happened in this case, but we should keep an eye on it.
